src/cameraGeom/Detector.cc

```diff
@@ -101,6 +101,18 @@ std::shared_ptr<geom::TransformPoint2ToPoint2> Detector::getTransform(FromSysT c
     return _transformMap.getTransform(makeCameraSys(fromSys), makeCameraSys(toSys));
 }
 
+template <typename FromSysT, typename ToSysT>
+geom::Point2D Detector::transform(geom::Point2D const &point, FromSysT const &fromSys,
+                                  ToSysT const &toSys) const {
+    return _transformMap.transform(point, makeCameraSys(fromSys), makeCameraSys(toSys));
+}
+
+template <typename FromSysT, typename ToSysT>
+std::vector<geom::Point2D> Detector::transform(std::vector<geom::Point2D> const &points,
+                                               FromSysT const &fromSys, ToSysT const &toSys) const {
+    return _transformMap.transform(points, makeCameraSys(fromSys), makeCameraSys(toSys));
+}
```

Comment:
I'm surprised these aren't defined in the header, since they're so simple.

Reply:
I prefer to define functions in the .cc file unless there's a good reason to do otherwse (e.g. inlining a method. It reduces clutter in the header saves on compilation time if something changes.